assets/templatesDocumentation/python3Console/README.md

```diff
@@ -0,0 +1,27 @@
+
+# Python 3 Console template
+
+How to create a virtual environment on this template:
+
+ - Press F1
+ - Select the command "Python: Create Environment"
+ - Select the environment type (venv or conda)
+ - Select a Python interpreter (this will be the version installed inside the
+ venv)
+ - If it doesn't select the Python from the venv, press F1 and select the command
+ "Developer: Reload Window")
+
+To install a Python package on the venv (and also on the Debug image), put it 
+on the "requirements-debug.txt" file (to install it on the Release image put it
+on the "requirements.release.txt" file)
+
+For more details, please watch the demonstration video below:
+
+<video controls>
+  <source src="../common/python3/createEnvPython3.mp4" type="video/mp4">
```

Comment:
This is returning `Error 401`, to solve this we will host the files on https://github.com/toradex/torizon-experimental-torizon-ide-v2-docs to not leave the repo here overloaded.